homeassistant/components/smartthings/__init__.py

```diff
@@ -25,10 +25,11 @@
     TOKEN_REFRESH_INTERVAL)
 from .smartapp import (
     setup_smartapp, setup_smartapp_endpoint, smartapp_sync_subscriptions,
-    validate_installed_app)
+    unload_smartapp_endpoint, validate_installed_app,
+    validate_webhook_requirements)
 
-REQUIREMENTS = ['pysmartapp==0.3.1', 'pysmartthings==0.6.7']
-DEPENDENCIES = ['webhook']
+REQUIREMENTS = ['pysmartapp==0.3.2', 'pysmartthings==0.6.7']
+DEPENDENCIES = ['cloud', 'webhook']
```

Comment:
We shouldn't add it as a dependency. The `async_is_active_subscription` will also check if it's logged in.